cpp/src/arrow/sparse_tensor_test.cc

```diff
@@ -49,7 +49,10 @@ static inline void AssertCOOIndex(const std::shared_ptr<Tensor>& sidx, const int
   }
 }
 
-TEST(TestSparseCOOIndex, Make) {
+//-----------------------------------------------------------------------------
```

Comment:
Perhaps there should also be some tests with empty/all-zero tensors?

Reply:
I will add these tests.